Tools/cases_generator/generate_cases.py

```diff
@@ -1257,13 +1286,13 @@ def always_exits(lines: list[str]) -> bool:
     )
 
 
-def variable_used(node: parser.Node, name: str) -> bool:
-    """Determine whether a variable with a given name is used in a node."""
+def variable_used(node: parser.Node, name: str, delim: str = None) -> bool:
+    """Determine whether a variable with a given name is used in a node.
+    """
```

Comment:
Revert completely to original.
```suggestion
def variable_used(node: parser.Node, name: str) -> bool:
    """Determine whether a variable with a given name is used in a node."""
```